webservice/src/main/resources/app/scripts/containers/Environment/EnvironmentContainer.jsx

```diff
@@ -152,7 +152,9 @@ class EnvironmentCards extends Component {
         <div className="service-box environment-box" data-id={namespace.id} ref={(ref) => this.nameSpaceRef = ref}>
           <div className="service-head clearfix">
             <h4 className="pull-left no-margin" title={namespace.name}>{Utils.ellipses(namespace.name, 15)}</h4>
-              <div className="pull-right">
+              {namespace.internal ?
+              '' :
```

Comment:
@joylyn instead of the blank string set it to null.

Reply:
@shahsank3t  Thanks for the review. I have made the change.